docs/advanced-configuration.md

```diff
@@ -80,6 +80,42 @@ splunkPlatform:
   logsEnabled: true
 ```
 
+## GKE Autopilot support
+
+If you want to run Splunk OTel Collector in [Google Kubernetes Engine
+Autopilot](https://cloud.google.com/kubernetes-engine/docs/concepts/autopilot-overview),
+make sure to set `distribution` setting to `gke/autopilot`:
+
+```yaml
+distribution: gke/autopilot
+```
+
+Sometimes Splunk OTel Collector agent daemonset can have [problems scheduling in
+Autopilot](https://cloud.google.com/kubernetes-engine/docs/concepts/daemonset#autopilot-ds-best-practices)
+If you run into these issues, you can assign the daemonset a higher [priority
+class](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/),
+this will make sure that the daemonset pods are always present on each node:
+
+1. Create a new priority class for Splunk OTel Collector agent:
+
+cat <<EOF | kubectl apply -f -
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
```

Comment:
Is this not something the distribution can handle by default with a flag to disable if desired?

Reply:
I started with providing another helm option for this, but not sure if `PriorityClass` is something we would like to manage in the helm chart. Also the issue doesn't happen often, and according to GKE folks it's something temporary, they are going to address it going forward OOTB. So I not sure we should introduce another help option because of this